Fibonacci.java

```diff
@@ -0,0 +1,29 @@
+import java.util.*;
+
+public class Fibonacci {
+  public static void main (String[] args) {
+    int fib_sum = 0;
+    int fib1 = 0;
+    int fib2 = 1;
+    boolean result = false;
+    System.out.println ("Enter any positive integer");
+    Scanner number = new Scanner ( System.in );
+    int InputNum = number.nextInt();

```

Comment:
This can throw exception if user input none-integer value. Best practice is catching everything you can, but if you have no idea what is exception - it's ok for now not to catch it.